cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -356,7 +357,7 @@ func IsEmptyStatus(cStorPool *apis.CStorPool) bool {
 		glog.Infof("cStorPool empty status: %v", string(cStorPool.ObjectMeta.UID))
 		return true
 	}
-	glog.Infof("Not empty status: %v", string(cStorPool.ObjectMeta.UID))
+	glog.Infof("Not empty status: %v, %v phase: %v", string(cStorPool.ObjectMeta.Name), string(cStorPool.ObjectMeta.UID), cStorPool.Status.Phase)
```

Comment:
the log does not convey meaning:
we can probably re-write this as below:
`cstor pool '%s': uid '%s': phase '%s': is_empty_status: false`

However, this is code smell. 
- It seems like debug logs.
- This function should be implemented purely as a predicate without any concern about logging/debugging.

Can you raise a issue to implement predicate for cstor-pool-mgmt at below package:
`pkg/cstorpool/v1alpha2/`

Reply:
Created the issue to have a predicate package in maya https://github.com/openebs/maya/issues/1009